test/README.md

```diff
@@ -1,13 +1,27 @@
 # E2E Tests
 
-This directory contains E2E tests which are run against BotKube installed on Kubernetes cluster and Slack API.
+This directory contains E2E tests. The tests instrument both Slack and Discord using a tester app and tester bot respectively.
+
+Basically, our testers listen to events sent from BotKube in a test cluster. And, the testers also trigger commands for BotKube to execute.
+
+On Kubernetes, the E2E tests are self-contained. They just require a BotKube installation on a cluster as highlighted in the instructions below. 
 
 ## Prerequisites
```

Comment:
Just a general comment for doc, wouldn't it be better to explain slack and discord separately? With current notation, even I haven't introduced slack changes, I will need to prepare slack-related things.  This can be minor but just wanted to prompt for the simplicity

Reply:
I think I need more clarification here...

- Are you referring to the developer adding tests that specifically target Slack and not discord?
- Or that the setup docs force you to configure both Slack and Discord - even though you might only want to test one of them?

Reply:
Option 2

Reply:
Fixed.